services/__tests__/country.spec.js

```diff
@@ -0,0 +1,171 @@
+import { jest } from '@jest/globals';
+
+import { dataRequest } from 'utils/request';
+
+import {
+  getCountriesProvider,
+  getRegionsProvider,
+  getSubRegionsProvider,
+  getFAOCountriesProvider,
+  getCountryLinksProvider,
+  getCategorisedCountries,
+  getCountryLinksSerialized,
+  GADM_DATASET,
+  SQL_QUERIES,
+} from 'services/country';
+import countryLinks from 'services/country-links.json';
+
+jest.mock('utils/request', () => ({
+  dataRequest: {
+    get: jest.fn(),
+  },
+}));
+
+jest.mock('services/country-links.json', () => ({
+  rows: [
+    {
+      iso: 'CMR',
+      external_links:
+        '[{  "title": "Interactive Forest Atlas of Cameroon",  "url": "http://cmr.forest-atlas.org/"}]',
+    },
+    {
+      iso: 'CAF',
+      external_links:
+        '[{ "title": "Interactive Forest Atlas of Central African Republic",  "url": "http://caf.forest-atlas.org/"}]',
+    },
+  ],
+}));
+
+describe('country service', () => {
```

Comment:
Love these tests, Will! 💪
Your descriptions are very detailed and really contribute to the value of these tests.